STABLE_RELEASES.md

```diff
@@ -0,0 +1,147 @@
+# Stable Releases
+
+The following release series are currently supported and receieve bugfixes:
+
+* **0.37**
+* **0.39 "Launchpad"**
+
+Stable releases continue to receieve bugfixes until they reach End Of Life.
+The `0.37.x` release series will continue receiving bugfixes until the Cosmos Hub
+migrates to a newer release of the Cosmos-SDK.
+
+The `0.38.x` release series will reach End of Life once the `0.39.0` release is published.
+
+## Point Releases
+
+Once a Cosmos-SDK release has been completed and published, updates for it are released under certain circumstances
+and must follow the [Point Release Procedure](CONTRIBUTING.md).
+
+## Rationale
+
+Unlike in-development `master` branch snapshots, **Cosmos-SDK** releases are subject to much wider adoption,
+and by a significantly different demographic of users. During development, changes in the `master` branch 
+affect SDK users, application developers, early adopters, and other advanced users that elect to use
+unstable experimental software at their own risk.
+
+Conversely, users of a stable release expect a high degree of stability. They build their applications on it, and the
+problems they experience with it could be potentially highly disruptive to their projects.
+
+Stable release updates are recommended to the vast majority of developers, and so it is crucial to treat them
+with great caution. Hence, when updates are proposed, they must be accompanied by a strong rationale and present
+a low risk of regressions, i.e. even one-line changes could cause unexpected regressions due to side effects or
+poorly tested code. We never assume that any change, no matter how little or non-intrusive, is completely exempt
+of regression risks.
+
+Therefore, the requirements for stable changes are different than those that are candidates to be merged in
+the `master` branch. When preparing future major releases, our aim is to design the most elegant, user-friendly and
+maintainable SDK possible which often entails fundamental changes to the SDK's architecture design, rearranging and/or
+renaming packages as well as reducing code duplication so that we maintain common functions and data structures in one
+place rather than leaving them scattered all over the code base. However, once a release is published, the
+priority is to minimise the risk caused by changes that are not strictly required to fix qualifying bugs; this tends to
+be correlated with minimising the size of such changes. As such, the same bug may need to be fixed in different
+ways in stable releases and `master` branch.
+
```

Comment:
it might be simpler to state that master is the development branch. 

God Speed to anyone using master! does anyone actually use master in production tho?

Reply:
I tried to use always `master` instead of the `current development branch` phrase. I assumed people know that `master` is our development branch but I am happy to reconsider/clarify further if you think we should do so.